.github/workflows/java-build-push-test.yaml

```diff
@@ -86,14 +86,15 @@ jobs:
           description: "Publishing..."
 
       - name: Inspect image to see if it already exists
-        id: inspect_image
-        continue-on-error: true
+        id: should_publish
         run: |
-          docker manifest inspect 493416687123.dkr.ecr.us-east-1.amazonaws.com/services/oss/modeldb/backend:${{ steps.image_info.outputs.tag }}
+          TARGETS=""
+          docker manifest inspect 493416687123.dkr.ecr.us-east-1.amazonaws.com/services/oss/modeldb/backend:${{ steps.image_info.outputs.tag }} || TARGETS="modeldb"
+          echo "targets=${TARGETS}" >> $GITHUB_OUTPUT
```

Comment:
what's TARGETS used for?

Reply:
In this repo just indicating whether to publish the single image. 
But in services it specifies which images to build (since they might not all succeed in publishing).